File: AbtinFarCalendar/Abtinfar_VCSample/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace Abtinfar_VCSample
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void AbtinFar1_ArabicDateChanged(string ArabicDate)
        {
            MessageBox.Show(ArabicDate);
        }

        private void AbtinFar1_EnglishDateChanged(string EnglishDate)
        {
            MessageBox.Show(EnglishDate);
        }

        private void AbtinFar1_DateChanged(string FDate)
        {
            MessageBox.Show(FDate);
        }

        private void AbtinFar1_DayEvents(string Events)
        {
            MessageBox.Show(Events);
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            ComboBox1.SelectedIndex = 0;
        }

        private void Button1_Click(object sender, EventArgs e)
        {
            AbtinFar1.TodayDate();
        }

        private void ComboBox1_SelectedIndexChanged(object sender, EventArgs e)
        {
            AbtinFar1.SetCalendarStyle(ComboBox1.SelectedItem.ToString());
        }

        private void Button2_Click(object sender, EventArgs e)
        {
            tharf.Text = AbtinFarConvertor1.ConvertDigits(Double.Parse(tadad.Text));
        }

        private void Button3_Click(object sender, EventArgs e)
        {
            txtShamsi01.Text = AbtinFarConvertor1.English2Persian(DateTime.Parse(txtMiladi01.Text));
        }

        private void Button4_Click(object sender, EventArgs e)
        {
            txtMiladi02.Text = AbtinFarConvertor1.Persian2English(txtShamsi02.Text).ToString();
        }

        private void Button5_Click(object sender, EventArgs e)
        {
            txtQamari.Text = AbtinFarConvertor1.Persian2Qamari(txtShamsi03.Text);
        }
    }
}

File: AbtinFarCalendar/Abtinfar_VBSample/Form1.vb
Public Class Form1
 Private Sub AbtinFar1_ArabicDateChanged(ArabicDate As String) Handles AbtinFar1.ArabicDateChanged
        MessageBox.Show(ArabicDate)
    End Sub
    Private Sub AbtinFar1_EnglishDateChanged(EnglishDate As String) Handles AbtinFar1.EnglishDateChanged
        MessageBox.Show(EnglishDate)
    End Sub

    Private Sub AbtinFar1_DayEvents(Events As String) Handles AbtinFar1.DayEvents
        MessageBox.Show(Events)
    End Sub

    Private Sub AbtinFar1_DateChanged(FDate As String) Handles AbtinFar1.DateChanged
        MessageBox.Show(FDate)
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        AbtinFar1.TodayDate()
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
       tharf.Text= AbtinFarConvertor1.ConvertDigits(Double.Parse(tadad.Text))
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        txtShamsi01.Text=AbtinFarConvertor1.English2Persian(DateTime.Parse(txtMiladi01.Text))
    End Sub

    Private Sub Button4_Click(sender As Object, e As EventArgs) Handles Button4.Click
        txtMiladi02.Text=AbtinFarConvertor1.Persian2English(txtShamsi02.Text)
    End Sub

    Private Sub Button5_Click(sender As Object, e As EventArgs) Handles Button5.Click
        txtQamari.Text=AbtinFarConvertor1.Persian2Qamari(txtShamsi03.Text)
    End Sub

    Private Sub ComboBox1_SelectedIndexChanged(sender As Object, e As EventArgs) Handles ComboBox1.SelectedIndexChanged
         AbtinFar1.SetCalendarStyle(ComboBox1.SelectedItem)
    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        ComboBox1.SelectedIndex=0
    End Sub

   
End Class
